scout/server/blueprints/variant/controllers.py

```diff
@@ -144,6 +145,19 @@ def variant(
     if add_case:
         variant_case(store, case_obj, variant_obj)
 
+    # Fetch ids for grouped cases and prepare alignment dispay
+    case_groups = {}
+    case_group_label = {}
+    if case_obj.get("group"):
+        for group in case_obj.get("group"):
+            case_groups[group] = list(store.cases(group=group))
+            case_group_label[group] = store.case_group_label(group)
+            LOG.debug("Populate aligments for entire case group {}:".format(group))
+            for grouped_case in case_groups[group]:
+                LOG.debug("Populate case {}".format(grouped_case))
+                case_append_alignments(grouped_case)
+                LOG.debug("Ok: {}".format(grouped_case))
```

Comment:
this is producing a lot of unused text, can we remove it?

![image](https://user-images.githubusercontent.com/28093618/109767617-19ce7a00-7bf8-11eb-9c7f-3cb5a0232f49.png)